terraform/environments/uat.env

```diff
@@ -4,3 +4,4 @@ export BUCKET=podaac-services-uat-terraform
 export TF_VAR_container_image_tag=develop
 export TF_VAR_active_profiles="[\"\"]"
 export TF_VAR_tea_mapping="{\"podaac-swot-uat-swodlr-protected\"=\"archive.swot.podaac.uat.earthdata.nasa.gov\"}"
+export TF_VAR_edl_jwks_path=".well-known/edl_ops_jwks.json"
```

Comment:
should this be
```suggestion
export TF_VAR_edl_jwks_path=".well-known/edl_uat_jwks.json"
```

Reply:
Every environment currently authenticates on OPS